docs/concepts/jcloud/yaml-spec.md

```diff
@@ -250,6 +250,45 @@ Below are the defaults and requirements for the configurations:
 
 After JCloud deployment using the autoscaling configuration, the Flow serving part is just the same; the only difference you may notice is it takes a few extra seconds to handle the initial requests since it needs to scale the deployments behind the scenes. Let JCloud handle the scaling from now on, and you should only worry about the code!
 
+## Configure Availability Tolerance
+
+In case of any unlikely event that would cause the service disruption of executors, JCloud allows users to specify a tolerance level in terms of number of replicas.
+
+The JCloud parameters minAvailable and maxUnavailable ensures the absense of some replicas of an executor to be with in the limits of toleration.
```

Comment:
```suggestion
The JCloud parameters `minAvailable` and `maxUnavailable` ensure that Executors will stay up even if a certain number of replicas go down.
```
Please spell-check in future :)

I did quite a significant rewrite. Is it still accurate?